safety/scan/render.py

```
@@ -179,80 +152,93 @@ def print_fixes_section(console: Console, requirements_txt_found: bool = False,
+            print_summary(console, unique_issues, 10, 2, project_model, dependencies_count=5, fixes_count=2)
+
     """
+    
     from ..util import pluralize
 
-    if is_detailed_output:
-        if not ignored_vulns_data:
-            ignored_vulns_data = iter([])
+    # Set the policy message based on the project source
+    if project.policy:
+        policy_msg = f"policy fetched from Safety Platform" if project.policy.source is PolicySource.cloud else f"local {project.id or 'scan policy file'} project scan policy"
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)